### ISSUE TITLE
Support for Missings

### ISSUE THREAD

**Issue by cstjean** [ajkeller34/Unitful.jl#188]
We're transitioning to `missing` values in our code base, instead of `NaN`, and find that there are some... missing... methods, like `uconvert(::Unitful.FreeUnits, ::Missing)`. I assume that it just hasn't been implemented yet? If so, I'll make a PR.

**Comment by ajkeller34** [comment]
`Missing` support sounds like a good idea but I haven't been following along with the latest thinking about it. What would you propose this method should do? Just return `missing`?

**Comment by cstjean** [comment]
Yeah. We already have that `1u"m" * missing === missing`, because it's a `Number`.

**Comment by ajkeller34** [comment]
Sounds good, thanks!

**Comment by nicoleepp** [comment]
Additionally methods like:

```julia
*(y::Missing, x::T) where {T<:Units} = missing
ustrip(x::Missing) = missing
```

would be good to have

**Comment by rafaqz** [comment]
Any idea of  the set of methods needed for a complete Unitful missings PR?

**Comment by cstjean** [comment]
I've been using 

```julia
Unitful.uconvert(::Any, ::Missing) = missing
Unitful.ustrip(::Missing) = missing
Unitful.unit(::Type{Union{Missing, T}}) where T = unit(T)
Base.:*(::Missing, ::Unitful.Units) = missing
Base.:/(::Missing, ::Unitful.Units) = missing
```

and it works fine. It has been heavily tested. I just haven't had the time to make a PR out of it. If anyone wants to take it, please do.

**Comment by rafaqz** [comment]
Also
```julia
Base.://(::Missing, ::Unitful.Units) = missing
Base.:^(::Unitful.Units, ::Missing) = missing
```

**Comment by rafaqz** [comment]
Should `unit(x::Missing)` return `missing` or `NoUnit`?

The same question also applies to `absoluteunit`, and`dimension(::Missing)` returning `missing` or `NoDims`

**Comment by cstjean** [comment]
`missing`. Otherwise `unit.([1.0m, missing, 3.3m])` would be strange.

**Comment by ajkeller34** [comment]
Closed by #208
